lib/src/network/url_tools.dart

```diff
@@ -0,0 +1,51 @@
+class MistralAPIEndpoints {
```

Comment:
no imports here?

Reply:
Nope, at least I don't see anything that should be imported